walkthroughs/week-5-tdd/intro/src/main/java/com/google/sps/Greeter.java

```diff
@@ -22,6 +22,7 @@
    * Returns a greeting for the given name.
    */
   public String greet(String name) {
+      name = name.replaceAll(" ","");
```

Comment:
What if my name is "Anna Beth"? This will now say "Hello AnnaBeth" instead of "Hello Anna Beth". Is that behavior that you want?

Reply:
Fixed. Thanks for the feedback Anna